### PR TITLE
Upgrade addon blueprint

### PR DESCRIPTION
**Pull request by scalvert**
Upgrades to the 2.15.1 blueprint.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 6,
+    ecmaVersion: 2017,
```
Comment (hjdivad) [inline .eslintrc.js]: the years just fly by

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
-  directories:
-    - $HOME/.npm
-    - $HOME/.cache # includes bowers cache
+  yarn: true
 
 env:
-  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
+  # we recommend new addons test the current and previous LTS
+  # as well as latest stable release (bonus points to beta/canary)
+  - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (hjdivad) [inline .travis.yml]: ember 2.8 is not supported https://github.com/hjdivad/ember-m3#requirements

File: config/ember-try.js [hunk 3]
```diff
@@ -2,29 +2,13 @@
 module.exports = {
   scenarios: [
     {
-      name: 'ember-2.13.4',
+      name: 'ember-lts-2.8',
```
Comment (hjdivad) [inline config/ember-try.js]: minimum is 2.10

File: config/ember-try.js [hunk 4]
```diff
@@ -2,29 +2,13 @@
...
       bower: {
         dependencies: {
-          'ember': 'components/ember#2.13.4'
+          'ember': 'components/ember#lts-2-8'
```
Comment (hjdivad) [inline config/ember-try.js]: as above

File: config/ember-try.js [hunk 5]
```diff
@@ -2,29 +2,13 @@
...
-    },
-    {
-      name: 'ember-2.14.1',
-      bower: {
-        dependencies: {
-          'ember': 'components/ember#2.14.1'
-        },
-        resolutions: {
-          'ember': '2.14.1'
+          'ember': 'lts-2-8'
```
Comment (hjdivad) [inline config/ember-try.js]: as above

File: .eslintrc.js (hunk 1 above)
Comment (scalvert) [reply .eslintrc.js]: indeed

**Pull request merged**